### PR TITLE
Fixed emoji insert at beginning

### PR DESCRIPTION
**Pull request by CepiPerez**
- Fixed missing signal for music browser

### CODE REVIEW

File: src/client/ui.py [hunk 1]
```diff
@@ -115,6 +115,7 @@ def initConnections(self,store):
 		self.rootObject().playRecording.connect(self.playRecording)
 		self.rootObject().deleteRecording.connect(self.deleteRecording)
 		self.rootObject().breathe.connect(self.onProcessEventsRequested)
+		self.rootObject().browseFiles.connect(self.browseFiles)
```
Comment (tgalal) [inline src/client/ui.py:118]: invalid signal

File: src/client/ui.py [hunk 2]
```diff
@@ -141,6 +142,8 @@ def initConnections(self,store):
 		self.rootObject().loadMessages.connect(self.messageStore.loadMessages);
 		
 		
+		self.messageStore.conversationMedia.connect(self.rootObject().onConversationMedia)
```
Comment (tgalal) [inline src/client/ui.py:145]: invalid slot

File: src/client/ui.py [hunk 3]
```diff
@@ -141,6 +142,8 @@ def initConnections(self,store):
...
+		self.rootObject().getConversationMediaByJid.connect(self.messageStore.getConversationMedia)
```
Comment (tgalal) [inline src/client/ui.py:146]: invalid signal

**Comment by tgalal** [comment]
I only merged in emoji fix

File: src/client/ui.py (hunk 3 above)
Comment (CepiPerez) [inline src/client/ui.py:146]: sorry, I was trying Bojan dev and forgot to remove this line

**Pull request closed without merging**